plaidml/bridge/keras/backend_test.py

```diff
@@ -1515,30 +1515,33 @@ def testReverse(self, b, x, ax):
          ]],
         skip_theano=False,
         skip_tensorflow=True)
-    @unittest.skip("gather is not yet implemented")
     def testGather(self, b, v):
         I = b.variable(np.array([0, 2, 1, 0], dtype='int32'), dtype='int32')
-        I2 = b.variable(np.array([[2, 1], [0, 1], [1, 0], [2, 1], [0, 0]], dtype='int32'),
-                        dtype='int32')
+        #I2 = b.variable(np.array([[2, 1], [0, 1], [1, 0], [2, 1], [0, 0]], dtype='int32'),
```

Comment:
If I2 is unused (and apparently was previously as well), let's delete it rather than just commenting it out.